versions/3.1.0.md

```diff
@@ -2336,6 +2336,18 @@ As such, inline schema definitions, which do not have a given id, *cannot* be us
 The [xml](#schemaXml) property allows extra definitions when translating the JSON definition to XML.
 The [XML Object](#xmlObject) contains additional information about the available options.
 
+###### Picking Schema Vocabularies
+
+OpenAPI documents are made up of multiple resources, sometimes resources are all in the same root OpenAPI document (inline resources), or resources are contain their own documents with `$ref` (separate resources). It is important for tooling to be able to detect what meta-schema any given resource wishes to be processed with: JSON Schema Core, JSON Schema Validation, OpenAPI Schema Object, or some custom meta schema.
+
+The `$schema` keyword can be used to define which meta-schema should be used when interacting with any resource.
```

Comment:
Again, only valid within a valid JSON Schema resource or a schema-object resource or within schema-objects within an OAS sub-document containing a `components` object.

Reply:
Would you like to take a swing at improving my nonsense with the Suggestion feature? I would appreciate the help.

Reply:
I think I might have got this sorted.